modules/virt-schedule-best-match-cpu-model-vms.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-scheduling-vms.adoc
+
+[id="virt-schedule-best-match-cpu-model-vms_{context}"]
+= Scheduling virtual machines with the best match for CPU model
+
+When a CPU model is not defined for a node and a virtual machine (VM), the CPU model for the VM is set to `host-model` and the VM is scheduled on the node that offers the best match for the VM's CPU model.
+
+.Prerequisites
+* You configured CPU node discovery.
```

Comment:
This is done automatically. It is part of the default CNV deployment.

No need to do it.

Reply:
Removed the prerequisite for configuring CPU node discovery because it is enabled.